guides/v2.2/extension-dev-guide/plugins.md

```diff
@@ -137,7 +137,7 @@ class ProductAttributesUpdater
 }
 ```
 
-After methods have access to all the arguments of their observed methods. When the observed method completes, Magento passes the result and arguments to the next after method that follows. If observed method does not return a result (`@return void`), then it passes `null` to the next after method.
+The after methods have access to all the arguments of their observed methods. When the observed method completes, Magento passes the result and arguments to the next after method that follows. If the observed method does not return a result (`@return void`), then it passes a `null` value to the next after method.
```

Comment:
Change to:
`_after_ methods`

Reply:
I disagree with this. I don't know why after is italicized above. There are plenty of instances where before, after, and around are not italicized. I would remove the italics above.

Reply:
I changed all before, after, and around formatting to inline code for consistency.

Reply:
So, typically one would emphasize a first occurrence for a definition. After that, it should not be emphasized. When it is used as an inline code, that is referring to the method in the code.